test/test_type.jl

```diff
@@ -11,6 +11,20 @@ isdir(outdir) && rm(outdir, recursive = true)
 
 # Run unit tests for various equations
 @testset "Test Type Stability" begin
+    @timed_testset "mean values" begin
+        for RealT1 in (Float32, Float64), RealT2 in (Float32, Float64)
+            RealT = promote_type(RealT1, RealT2)
+            @test typeof(@inferred Trixi.ln_mean(RealT1(1), RealT2(2))) == RealT
+            @test typeof(@inferred Trixi.inv_ln_mean(RealT1(1), RealT2(2))) == RealT
+            for RealT3 in (Float32, Float64)
+                RealT = promote_type(RealT1, RealT2, RealT3)
+                @test typeof(@inferred Trixi.stolarsky_mean(RealT1(1), RealT2(2),
+                                                            RealT3(3))) ==
+                      RealT
```

Comment:
```suggestion
                @test typeof(@inferred Trixi.stolarsky_mean(RealT1(1), RealT2(2),
                                                            RealT3(3))) == RealT
```
Is this possible? Or is this ugliness due to the formatter?

Reply:
That's what the formatter wants to have 😢

Reply:
Oh well, no worries. We are at its mercy sometimes like with the strange formatting [here](https://github.com/trixi-framework/TrixiShallowWater.jl/blob/main/src/equations/shallow_water_exner_1d.jl#L248).